Sources/UtiliKit/General/UISegmentedControl+Extensions.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  UISegmentedControl+Extensions.swift
+//  UtiliKit-iOSTests
```

Comment:
Very minor: The module name in the comment doesn't match the module this belongs to.
```suggestion
//  UtiliKit-iOS
```